docs/source/tutorials/hospital_admissions_model.qmd

```diff
@@ -22,33 +22,35 @@ $$
 h(t) \sim \text{HospDist}\left(H(t)\right)
 $$
 
-Where $h(t)$ is the observed number of hospital admissions at time $t$, and $H(t)$ is the number of latent hospital admissions at time $t$. The distribution $\text{HospDist}$ is discrete. For this example, we will use a negative binomial distribution:
+Where $h(t)$ is the observed number of hospital admissions at time $t$, and $H(t)$ is the number of latent hospital admissions at time $t$. The distribution $\text{HospDist}$ is discrete. For this example, we will use a negative binomial distribution with an inferred concentration.
 
 $$
 \begin{align*}
-h(t) & \sim \text{NegativeBinomial}\left(\text{concentration} = 1, \text{mean} = H(t)\right) \\
-H(t) & = \omega(t) p_\mathrm{hosp}(t) \sum_{\tau = 0}^{T_d} d(\tau) I(t-\tau)
+h(t) & \sim \mathrm{NegativeBinomial}\left(\mathrm{mean} = H(t), \mathrm{concentration} = k\right) \\
+H(t) & = p_\mathrm{hosp}(t) \sum_{\tau = 0}^{T_d} d(\tau) I(t-\tau) \\
+\log[p_\mathrm{hosp}(t)] & \sim \mathrm{Normal}(\mu=\log(0.05), \sigma=\log(1.1) \\
+\log(k) & \sim \mathrm{Normal}(\mu=\log(1), \sigma=\log(10))
```

Comment:
any particular reason for using $log$ when defining the standard deviation?

Reply:
Makes it easier to compute the implications of the 68/95 rule on the natural scale.

$\exp\Big(\log(M) - \log(S)\Big)$ = M / S$

$\exp\Big(\log(M) + 2 * \log(S)\Big) = MS^2$

etc.